configuration/backend_configuration/jsonkeyvalues/patientIdPreportConfig.json

```diff
@@ -0,0 +1,18 @@
+{
+    "report.patientIdSticker.fields.identifier": "true",
+    "report.patientIdSticker.fields.name": "true",
+    "report.patientIdSticker.fields.dob": "true",
+    "report.patientIdSticker.fields.age": "true",
+    "report.patientIdSticker.fields.gender": "true",
+    "report.patientIdSticker.fields.fulladdress": "true",
+    "report.patientIdSticker.fields.label.font.size": "2",
+    "report.patientIdSticker.fields.label.value.font.size": "4",
+    "report.patientIdSticker.fields.label.gap": "1mm",
+    "report.patientIdSticker.stylesheet": "patientIdStickerFopStylesheet.xsl",
+    "report.patientIdSticker.logourl": "web/module/resources/OpenMRS_logo.png",
```

Comment:
Is this logo any different from the one in openmrs core?

Reply:
thank you @dkayiwa 
I have removed the logo config since the openMRS logo will be loaded by default based on [this](https://github.com/openmrs/openmrs-module-patientdocuments/pull/3)